tests/UI/PresenterComponentReflection.combineArgs.phpt

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Test: PresenterComponentReflection::combineArgs()
+ */
+
+use Nette\Application\UI\PresenterComponentReflection;
+use Tester\Assert;
+
+require __DIR__ . '/../bootstrap.php';
+
+class MyPresenter
+{
+
+	public function myMethod($intParam = 0, $strParam = '')
+	{
+	}
+
+}
+
+$reflection = Nette\Utils\Callback::toReflection(Nette\Utils\Callback::check(['MyPresenter', 'myMethod']));
```

Comment:
Can you change it to simpler `$reflection = new ReflectionMethod('MyPresenter', 'myMethod');` ?